sql/addons_v3.sql

```diff
@@ -0,0 +1,14 @@
+SELECT
+  @submission_date AS submission_date,
```

Comment:
based on the discussion [here](https://github.com/mozilla/bigquery-etl/pull/41#discussion_r269596130) I think we should do this to ease backfill:

```suggestion
  submission_date_s3 AS submission_date,
```